schema.development.kf

```diff
@@ -330,6 +294,75 @@ action remove_credential($id) public {
     );
 }
 
+action share_credential(
+        $id,
+        $original_credential_id,
+        $credential_type,
+        $credential_level,
+        $credential_status,
+        $content,
+        $encryption_public_key) public {
+    $converted = idos_near.implicit_address_to_public_key(@caller);
+    SELECT CASE
+    WHEN NOT EXISTS (
+        SELECT 1 from credentials
+            WHERE id = $original_credential_id
+            AND human_id=(SELECT DISTINCT human_id FROM wallets WHERE (wallet_type = 'EVM' AND address=@caller COLLATE NOCASE)
+                OR (wallet_type = 'NEAR' AND public_key = $converted))
+    ) THEN ERROR('The original credential does not belong to the caller') END;
+
+    add_credential($id, $credential_type, $credential_level, $credential_status, $content, $encryption_public_key);
+    INSERT INTO shared_credentials (original_id, duplicate_id) VALUES ($original_credential_id, $id);
+}
+
+action share_credential_as_issuer (
```

Comment:
Needs a better name. I can't figure out a good one off the top of my head, but here's a suggestion:
```suggestion
action share_credential_through_access_grant (
```